### PR TITLE
Truncate menu items

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: components/PageNavigation.vue [hunk 1]
```diff
@@ -272,6 +272,7 @@
       margin: 0 0 0.25rem 0;
       position: relative;
       margin-right: 0;
+      margin-bottom: 0;
```
Comment (LeoniePeters) [inline components/PageNavigation.vue:275]: I don't see why this is needed.
If it is, in that case you could just set margin: 0 and remove margin-right and margin-bottom.

File: components/PageNavigation.vue [hunk 2]
```diff
@@ -295,6 +296,16 @@
             color: $white;
           }
         }
+        span {
+          overflow: hidden;
+          white-space: nowrap;
+          display: block;
```
Comment (LeoniePeters) [inline components/PageNavigation.vue:302]: Remove this display: block. It overwrites the above (line 222) set display: flex which keeps the icon and text properly aligned and the element height correct.

File: components/PageNavigation.vue [hunk 3]
```diff
@@ -295,6 +296,16 @@
...
+          text-overflow: ellipsis;
+          height: 1.5rem;
```
Comment (LeoniePeters) [inline components/PageNavigation.vue:304]: Remove this height as it's unnecessary when display is set to flex as mentioned above.

File: components/PageNavigation.vue [hunk 4]
```diff
@@ -295,6 +296,16 @@
...
+          i {
+            margin-right: 0.5rem;
```
Comment (LeoniePeters) [inline components/PageNavigation.vue:306]: Is this needed?

**Review by LeoniePeters — CHANGES_REQUESTED** [review]

**Review by mirjamverloop — APPROVED** [review]

**Pull request merged**